pkg/securitypolicy/regopolicy_test.go

```diff
@@ -3957,6 +3957,75 @@ func Test_Rego_EnvListGetsRedacted(t *testing.T) {
 	}
 }
 
+func Test_Rego_Enforce_CreateContainer_ConflictingAllowStdioAccessHasErrorMessage(t *testing.T) {
+	constraints := generateConstraints(testRand, 1)
+	constraints.containers[0].AllowStdioAccess = true
+
+	// create a "duplicate" as far as create container is concerned excep for
+	// a different "AllowStdioAccess" value
+	duplicate := &securityPolicyContainer{
```

Comment:
you can do 
```go
duplicate := *constraints.containers[0]
duplicate.AllowStdioAccess = false
// ...
constraints.containers = append(constraints.containers, &duplicate)
```
to avoid needing to manually copying the individual fields, so that this wont need to be updated if fields are added/changed in `securityPolicyContainer`

Reply:
also, can you not use `constraints.containers[0].clone`, similar to what the external process test below does?

Reply:
this was intentionally done as not a clone to make clear that it is "like the other but not the same"